pkg/generated/openapi/cmd/models-schema/main.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/go-openapi/spec"
+
+	"k8s.io/kubernetes/pkg/generated/openapi"
+)
+
+// Outputs openAPI schema JSON containing the schema definitions in zz_generated.openapi.go.
```

Comment:
really?  I thought we already had this.

Reply:
Today, the best way to generate the openapi is to run an apiserver and get `/openapi/v2` (which is what we do in `hack/update-openapi-spec.sh`).

Reply:
`swagger.json` is a downstream artifact of this that combines this information with endpoint information to construct the complete OpenAPI.  This artifact only has information about the `definitions` (the types, not the endpoints). The problem is that depending directly on `swagger.json` introduces a dependency cycle. I put a section in the description summarizing.